docs/how-tos/using-the-event-bus.rst

```diff
@@ -10,17 +10,19 @@ Setup
 
 To start producing and consuming events using the Open edX Event Bus, follow these steps:
 
-Install the Open edX Event Bus Plugin
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Step 1: Install the Open edX Event Bus Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 First, you need to install the Open edX Event Bus plugin in both the producing and consuming services. The plugin is a Django app that provides the necessary tools and configurations to produce and consume events. You could install the Redis plugin by running:
 
 .. code-block:: bash
 
    pip install edx-event-bus-redis
 
-Configure the Event Bus
-~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: Redis is the community-supported plugin for the Open edX Event Bus and is the recommended plugin to use. You can find more information about the Redis plugin in the `event-bus-redis`_ repository. However, you can also implement your own plugin with your preferred message broker by following the :doc:`../how-tos/add-new-event-bus-concrete-implementation` documentation.
```

Comment:
I think that both redis and kafka are community supported, but redis is just the default. We should mention Kafka as an alternative before doing another implementation.

Reply:
Thanks for clarifying, I actually thought it was only Redis. I've changed the note with the suggestion: https://github.com/openedx/openedx-events/pull/426/commits/1a4bafa3526508656616ff569643dd764b69f0f6